flume-ng-sinks/flume-hdfs-sink/pom.xml

```diff
@@ -191,6 +191,37 @@ limitations under the License.
 
       </dependencies>
     </profile>
+
+    <profile>
+      <id>hbase-1.0.0</id>
+      <activation>
+        <property>
+          <name>hadoop.profile</name>
```

Comment:
Why did you use "hadoop.profile" instead of "hbase.profile" to
set with <value>hbase-1.0.0</value>

?